<html>
<head>
<title>PyErr_SetFromErrnoWithFilenameObject</title>
<meta name='robots' content='noindex,nofollow'>
<meta name='generator' content='GLOBAL-5.8.1'>
</head>
<body text='#191970' bgcolor='#f5f5dc' vlink='gray'>
<pre>
<a href='../S/2696.html#L667'>PyErr_SetFromErrnoWithFilenameObject</a>  667 Modules/posixmodule.c     rc = PyErr_SetFromErrnoWithFilenameObject(PyExc_OSError,
<a href='../S/2729.html#L468'>PyErr_SetFromErrnoWithFilenameObject</a>  468 Modules/zipimport.c         PyErr_SetFromErrnoWithFilenameObject(PyExc_IOError, key);
<a href='../S/3106.html#L433'>PyErr_SetFromErrnoWithFilenameObject</a>  433 Python/errors.c     PyObject *result = PyErr_SetFromErrnoWithFilenameObject(exc, name);
<a href='../S/3106.html#L445'>PyErr_SetFromErrnoWithFilenameObject</a>  445 Python/errors.c     PyObject *result = PyErr_SetFromErrnoWithFilenameObject(exc, name);
<a href='../S/3106.html#L454'>PyErr_SetFromErrnoWithFilenameObject</a>  454 Python/errors.c     return PyErr_SetFromErrnoWithFilenameObject(exc, NULL);
</pre>
</body>
</html>
